_topic_maps/_topic_map.yml

```diff
@@ -2740,23 +2763,6 @@ Topics:
   - Name: Configuring the Cluster Observability Operator to monitor a service
     File: configuring-the-cluster-observability-operator-to-monitor-a-service
 ---
-Name: Power monitoring
-Dir: power_monitoring
-Distros: openshift-enterprise,openshift-origin
```

Comment:
Here, Distros was needed because Power Monitoring was its own thing, but since its now part of Observability it doesn't require the Distro anymore.

Reply:
You're onto something here, but the books under the CICD directory do have Distros listed. Since the build works for Builds under CICD, I wonder if there is a different topic map / Distros issue going on here.

Reply:
@skrthomas great suggestion to check the heading IDs! All checks have passed. Thank you for helping me troubleshoot!

Reply:
You got it! I'm happy to help :)